### PR TITLE
add earth2observe & statista packages

### PR DESCRIPTION
**Pull request by MAfarrag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MAfarrag** [comment]
@conda-forge-linter @jezdez @bmabey can you please check this all checks have passed here

**Comment by MAfarrag** [comment]
@conda-forge-linter @jezdez @bmabey can you please check here all checks are passed

**Comment by MAfarrag** [comment]
@conda-forge-linter @haesleinhuepf  @jezdez  @bmabey  @ceball  can you please check here all checks are passed

**Comment by MAfarrag** [comment]
@hmaarrfk can you help here please ? ready for review more than over a week ago. Thanks

File: recipes/statista/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```
Comment (hmaarrfk) [inline recipes/statista/LICENSE.md:1]: i don't think you need this file do you? isn't it included in your github archive?

File: recipes/statista/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - loguru >=0.5.3
+    - pytest >=6.2.5
+    - pytest-cov >=2.12.1
+
+test:
+  imports:
+    - statista
+
+about:
+  home: https://github.com/MAfarrag/pyramids
```
Comment (hmaarrfk) [inline recipes/statista/meta.yaml:37]: This seems like the wrong url.

File: recipes/statista/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+    - python >=3.7.1,<3.10
+    - pip >=21.3.1
+    - matplotlib-base >=3.4.2,<3.6.0
+    - pandas >=1.3.2,<1.4.3
+    - scipy >=1.8.1
+    - scikit-learn >=0.24.2,<1.1.2
+    - loguru >=0.5.3
+    - pytest >=6.2.5
+    - pytest-cov >=2.12.1
```
Comment (hmaarrfk) [inline recipes/statista/meta.yaml:30]: depend on what you want, but it is very likely that you don't need `pytest`  to run your package. It seems to be a testing dependency. You might as well remove it from here.

File: recipes/statista/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
+  run:
+    - python >=3.7.1,<3.10
+    - pip >=21.3.1
+    - matplotlib-base >=3.4.2,<3.6.0
```
Comment (hmaarrfk) [inline recipes/statista/meta.yaml:24]:
```suggestion
    - numpy
    - matplotlib-base >=3.4.2,<3.6.0
```

your package seems to depend on numpy. You should depend on it
https://github.com/MAfarrag/statista/blob/main/statista/tools.py

File: recipes/earth2observe/LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```
Comment (hmaarrfk) [inline recipes/earth2observe/LICENSE.md:1]: Again, please confirm that ou need this here. You might as well just pull it from the package.

**Comment by MAfarrag** [comment]
@hmaarrfk Thanks for the review, now all are updated and passed. Thanks

**Comment by hmaarrfk** [comment]
Thank you!

**Pull request merged**